lib/helpers/DictionaryApiHelper.ts

```diff
@@ -12,7 +12,7 @@ export class DictionaryApiHelper {
     return await response.json();
   }
 
-  async exists(id: string) {
+  async doesExist(id: string) {
```

Comment:
It seems that we both have names for doesExist and exists in our test helpers. We need to align on which one we prefer.
what is your opinion on this??

Reply:
The exists method should also return a boolean. And as far as I can see we've done a mix of both recently 🙈. I prefer the exists() since there is no reason to add a "does" before in the methods as it is redundant.